hdl/modules/fofb_shaper_filt/cheby/wb_fofb_shaper_filt_regs.cheby

```diff
@@ -0,0 +1,69 @@
+memory-map:
+  bus: wb-32-be
+  name: wb_fofb_shaper_filt_regs
+  description: Interface to FOFB IIR shaper filters regs
+  x-hdl:
+    busgroup: true
+  children:
+    - repeat:
+        name: ch
+        count: 12
+        children:
+          - memory:
+              name: coeffs
+              memsize: 320
+              interface: sram
+              description: |
+                Coefficients for the ceil('max_filt_order'/2) IIR internal
+                biquads.
+
+                Each biquad takes 5 coefficients: b0, b1, b2, a1 and a2 (a0 = 1).
+                The 'coeffs' array should be populated in the following manner:
+
+                  coeffs[0 + 8*{biquad_idx}] = b0 of biquad {biquad_idx}
+                  coeffs[1 + 8*{biquad_idx}] = b1 of biquad {biquad_idx}
+                  coeffs[2 + 8*{biquad_idx}] = b2 of biquad {biquad_idx}
+                  coeffs[3 + 8*{biquad_idx}] = a1 of biquad {biquad_idx}
+                  coeffs[4 + 8*{biquad_idx}] = a2 of biquad {biquad_idx}
+                  coeffs[5 + 8*{biquad_idx}] = unused
+                  coeffs[6 + 8*{biquad_idx}] = unused
+                  coeffs[7 + 8*{biquad_idx}] = unused
+
+                NOTE: This ABI supports up to 20th order filters, but only the
+                coefficients corresponding to the first 'max_filt_order' filters
+                are meaningful for the gateware.
+              children:
+                - reg:
+                    name: val
+                    width: 32
+                    access: rw
+                    description: |
+                      Coefficient value using 'coeffs_fp_repr' fixed-point
+                      representation. It should be aligned to the left.
+    - reg:
+        name: max_filt_order
+        width: 32
+        access: ro
+        description: |
+          Maximum filter order supported by the gateware.
```

Comment:
I don't know if it's worth changing this at this point, but wouldn't it be simpler instead to specify `number_biquads`?

That way you don't need to specify any divisions or think about how many biquads you have with some `max_filt_order` value.

Reply:
Totally would! Thanks!